code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm

```diff
@@ -16,517 +16,23 @@
 /obj/item/weapon/reagent_containers/food/drinks/drinkingglass/on_reagent_change()
 	cut_overlays()
 	if (reagents.reagent_list.len > 0)
```

Comment:
All the more reason to encourage people to actually make sprites for the drinks 😏